File: cs50x/week6/sentimental-mario-more/mario.py
from cs50 import get_int

# Initialize height
height = 0

# Prompt user for height until valid input (1 to 8)
while height < 1 or height > 8:
    height = get_int("Height: ")

# Generate the pyramid
for i in range(1, height + 1):
    # Print spaces before the hashes for each row
    for j in range(height):
        if j < (height - i):
            print(" ", end="")  # Print spaces
        else:
            print("#", end="")  # Print left hashes

    # Print the gap and right hashes
    print(" ", "#" * i)

File: cs50x/week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main()
{
    int height_user;
    do
    {
        height_user = get_int("Height ");
    }
    while (height_user <= 0 || height_user >= 9);
    for (int height = 1; height <= height_user; height++)
    {
        for (int space = 0; space < height_user - height; space++)
        {
            printf(" ");
        }
        for (int line = 0; line < height; line++)
        {
            printf("#");
        }
        printf("  ");
        for (int line = 0; line < height; line++)
        {
            printf("#");
        }
        printf("\n");
    }
}
